platforms/ide/problems-api/build.gradle.kts

```diff
@@ -37,6 +37,7 @@ dependencies {
     implementation(libs.guava)
     implementation(libs.inject)
 
+    testImplementation(project(":logging"))
```

Comment:
❓ Is this needed? The contributed test seems to have no imports from the logging subproject.

Reply:
It's needed for `import org.gradle.internal.deprecation.Documentation`